addons/website_sale/models/product_image.py

```diff
@@ -41,3 +41,9 @@ def _check_valid_video_url(self):
         for image in self:
             if image.video_url and not image.embed_code:
                 raise ValidationError(_("Provided video URL for '%s' is not valid. Please enter a valid video URL.") % image.name)
+
+    def create(self, vals):
```

Comment:
tag it `@api.model_create_multi` for consistency/documentation, and rename `vals` to `vals_list`